package.json

```diff
@@ -1,33 +1,19 @@
 {
-  	"name": "tree-sitter-norg",
-  	"version": "0.1.0",
-  	"description": "A TreeSitter Parser For The NFF",
-  	"main": "bindings/node",
-  	"scripts": {
```

Comment:
These scripts are important as they're embedded in my muscle memory now :).
The only thing that got preserved was `yarn test`, we wanna add all the previous scripts back.

Reply:
i say we keep the package.json as it was before because there are bits of metadata that may be important to other editors that want to utilize this TS parser